### PR TITLE
remove hostname from zookeeper mntr data stream

### LINKED ISSUE #11600: [Zookeeper] Remove hostname field from mntr datastream
Following discussions on https://github.com/elastic/sdh-beats/issues/5316, the `hostname` field shouldn't be in the `mntr` dataset.

It can't be fetched from the `mntr` output during testing and it's not included in the Zookeper docs:
Image

The purpose of this issue is to remove it from the integration and double-check any other inconsistencies.

### PR DESCRIPTION
**Pull request by stefans-elastic**
## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).
- [ ] I have verified that any added dashboard complies with Kibana's [Dashboard good practices](https://docs.elastic.dev/ux-guidelines/data-viz/dashboard-good-practices) 

## Author's Checklist

- [ ] 

## How to test this PR locally
Run zookeeper locally
```
docker run -it -d -p 2181:2181 -p 2888:2888 -p 3888:3888 -e ZOO_4LW_COMMANDS_WHITELIST=srvr,mntr,cons,conf zookeeper
```

Add Zookeeper integration in local Kibana
Follow instructions of adding an agent
(to make it work locally I added following lines into my `/etc/hosts` file
```
127.0.0.1       fleet-server
127.0.0.1       elasticsearch
```

For generating some extra metrics I created a new key in zookeeper
```
docker exec -it <ZOOKEEPER_CONTAINER_ID> bash
./bin/zkCli.sh
create /test test-data
get /test
quit
exit
```
Check Kibana Discover page to make sure `zookeeper.mntr.hostname` field is not showing up
Screenshot 2024-11-22 at 2 28 40 PM

## Related issues

- Closes #11600

### CODE REVIEW

File: packages/zookeeper/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.13.1"
+  changes:
+    - description: Removed hostname field from mntr data stream
```
Comment (gpop63) [inline packages/zookeeper/changelog.yml:4]:
```suggestion
    - description: Remove `hostname` field from mntr data stream.
```

Comment (stefans-elastic) [reply packages/zookeeper/changelog.yml]: thanks @gpop63 . I've updated the message

**Comment by lalit-satapathy** [comment]
>https://github.com/elastic/integrations/pull/11832/commits/2e588f0a8e9b2e6fd212a5dadeb721fdf57f7906
>Stefan Stas committed 4 days ago

Commits not from proper user id/email.

**Comment by stefans-elastic** [comment]
> > [2e588f0](https://github.com/elastic/integrations/commit/2e588f0a8e9b2e6fd212a5dadeb721fdf57f7906)
> > Stefan Stas committed 4 days ago
> 
> Commits not from proper user id/email.

resolving...

**Pull request closed without merging**